### DISCUSSION TITLE
Why is @docusaurus/faster a prod dependency?

### DISCUSSION THREAD

**Discussion by IanVS** [facebook/docusaurus#10756] [Q&A] [answered]
The instructions for installing @docusaurus/faster add it to the `dependencies` of `package.json`, but it seems like it's really just a set of build tooling.  Am I missing something, and part of this ends up in the production bundle that I'll deploy, or is this safe to move to my `devDependencies`?

**Comment by slorber** [comment, marked as answer]
You can add it to any of those, it doesn't mater. 

I think you misunderstand the difference between dependencies/dev Dependencies for first-party apps, this is not related in any way to what's added to the JS bundle, and only affects what gets installed in your node_modules when running certain npm commands that you probably don't use today.

**Reply by IanVS** [reply]
Yes I understand it's somewhat of a distinction without a difference, however I find it cleaner to keep dependencies separated this way.  Thanks for the answer.

**Reply by slorber** [reply]
Docusaurus has no nodejs runtime (SSR) and only does SSG. So the "production environment" of a Docusaurus site is your CI. To me it looks clearer to put this dependency (that is required at build time) in prod dependencies. What you can put as dev dependencies is things that are not involved in the build, such as code and md linters and formatters, that you only run locally.
